java-frontend/src/main/java/org/sonar/java/se/checks/SECheck.java

```diff
@@ -103,7 +107,12 @@ public String getMessage() {
     }
 
     public Set<List<JavaFileScannerContext.Location>> getFlows() {
-      return flows;
+      Stream<Flow> flowsAsStream = flows.stream();
```

Comment:
Maybe bit more clearer like this
```
Set<List<JavaFileScannerContext.Location>> nonExceptionalFlows = flows.stream().filter(Flow::isNonExceptional).map(Flow::elements).collect(Collectors.toSet());
if (!nonExceptionalFlows.isEmpty()) {
    return nonExceptionalFlows;
}
return flows.stream().map(Flow::elements).collect(Collectors.toSet());
```
Also you can drop `isExceptional` method on `Flow` afterwards